### DISCUSSION TITLE
New management staff

### DISCUSSION THREAD

**Discussion by yanick** [testing-library/svelte-testing-library#187] [General]
So a few new peeps have been added as admins of this repo as of today. I didn't see any deployment / house rules docs anywhere, so I thought we might want to at least start a discussion to sync everybody up.

And... as a start, should we instate a rule that two admins must :+1: a PR before merging it? I'm saying that mostly to save the repo from potential bad side-effects of my eagerness.  :grin:

@benmonro @EmilTholin @mihar-22 @MirrorBytes @sebastianrothe @yanick

**Comment by EmilTholin** [comment]
Hey @yanick!

That sounds reasonable. I don't have much time for open source at all these days, so there might be a shortage of 👍 from me even though the PR looks good 😅

**Comment by benmccann** [comment]
You could probably just go ahead and merge all the Dependabot stuff to clear out the queue without another approval since there's not much to manually review on those PRs

**Reply by yanick** [reply]
Sounds good. Will do!

**Comment by mcous** [comment]
Hello everyone! I've exchanged a few emails with @yanick about this, so I figured I'd chime in here, too. Over at [Viam](https://github.com/viamrobotics), svelte-testing-library has become an important part of our UI-testing strategy. Accordingly, I wanted to devote some time and energy to help maintain the project, especially with Svelte v5 on the horizon.

I've started with low-hanging fruit by tagging a few issues that I think can be closed out, but I'm happy to help out wherever is most useful!

**Reply by mcous** [reply]
@yanick I've been helping out for a few months here, and I think it's been a fruitful collaboration! At this point, I was wondering if you'd be open to granting me write access to the repo?

I'd love to be better able to triage issues, and the completionist in me really wants to clean up the dependabot queue. It'd also make it easier for me to test out improvements to the CI workflows. Plus it'd insulate the project a little from the ups and downs of our work schedules.

Let me know what you think!

**Reply by yanick** [reply]
If it was one of my repos, I'd say I prefer to keep the ultimate power. But I am merely a de factor steward of this project, and I won't pull a Denethor on y'all. :-)  You indeed did a lot in the last few months, and it's fair that you get access to the backstage. Let's see if I have the permission to do that...

**Reply by yanick** [reply]
There we go. Enjoy responsibly your new powers. :-)
